src/android/IntentShim.java

```diff
@@ -185,6 +187,11 @@ else if (action.equals("onIntent"))
 
             this.onNewIntentCallbackContext = callbackContext;
 
+            if (this.deferredIntent != null) {
+                fireOnNewIntent(this.deferredIntent);
+                this.deferredIntent = null;
+            }
+
             PluginResult result = new PluginResult(PluginResult.Status.NO_RESULT);
             result.setKeepCallback(true);
             callbackContext.sendPluginResult(result);
```

Comment:
a call to sendPluginResult() was done on fireOnNewIntent()...should it be done here as well? It doesn't appear to impact anything that I am aware of.

Reply:
I am honestly not sure, if you've tested it for both the first launch and subsequent launches of the app (as described in the original issue) then I'd say it's probably fine